.eslintrc.json

```diff
@@ -21,16 +21,14 @@
     "rules": {
         // Turn on/off rules here as needed
 
-        "no-multiple-empty-lines":"off",
         "no-console":"off",
         "quotes": "off",
         "max-len": "off",
         "eol-last":"off",
-
```

Comment:
at this point we are doubling down and writing the same ones so we can probably start to git ignore it by now